components/latex_block/latex_block.test.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import LatexBlock from 'components/latex_block/latex_block';
+
+describe('components/LatexBlock', () => {
+    const defaultProps = {
+        content: '```latex e^{i\\pi} + 1 = 0```',
+        enableLatex: true,
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = shallow(<LatexBlock {...defaultProps}/>);
+
+        expect(wrapper).toMatchSnapshot();
+    });
```

Comment:
Can we add another test with `enableLatex` as false and one to make `katex.renderToString(content, katexOptions)` throw an error. That way we will see all the possible outputs in the snapshot